roslibrust/Cargo.toml

```diff
@@ -6,82 +6,49 @@ edition = "2021"
 license = "MIT"
 readme = "../README.md"
 description = "An library for interfacing with the ROS's rosbridge_server"
-repository = "https://github.com/Carter12s/roslibrust"
+repository = "https://github.com/roslibrust/roslibrust"
 keywords = ["ROS", "robotics", "websocket", "json", "async"]
 categories = ["science::robotics"]
 
 [dependencies]
-abort-on-drop = "0.2"
-anyhow = "1.0"
-byteorder = "1.4"
-dashmap = "5.3"
-deadqueue = "0.2.4" # .4+ is required to fix bug with missing tokio dep
-futures = "0.3"
-futures-util = "0.3"
-lazy_static = "1.4"
-log = "0.4"
-proc-macro2 = "1.0"
-rand = "0.8"
-serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
-smart-default = "0.6"
-thiserror = "1.0"
-tokio = { version = "1.20", features = [
-    "net",
-    "macros",
-    "time",
-    "rt-multi-thread",
-] }
-tokio-tungstenite = { version = "0.17" }
-uuid = { version = "1.1", features = ["v4"] }
-roslibrust_codegen_macro = { path = "../roslibrust_codegen_macro", version = "0.11.1" }
-roslibrust_codegen = { path = "../roslibrust_codegen", version = "0.11.1" }
-reqwest = { version = "0.11", optional = true } # Only used with native ros1
-serde_xmlrpc = { version = "0.2", optional = true } # Only used with native ros1
-roslibrust_serde_rosmsg = { version = "0.4", optional = true } # Only used with native ros1
-hyper = { version = "0.14", features = [
-    "server",
-], optional = true } # Only used with native ros1
-gethostname = { version = "0.4", optional = true } # Only used with native ros1
-regex = { version = "1.9", optional = true } # Only used with native ros1
-# TODO I think we should move rosapi into its own crate...
-serde-big-array = { version = "0.5", optional = true } # Only used with rosapi
+# TODO version here
+roslibrust_common = { path = "../roslibrust_common" }
+roslibrust_ros1 = { path = "../roslibrust_ros1", optional = true }
+roslibrust_rosbridge = { path = "../roslibrust_rosbridge", optional = true }
+roslibrust_zenoh = { path = "../roslibrust_zenoh", optional = true }
+roslibrust_mock = { path = "../roslibrust_mock", optional = true }
 
 [dev-dependencies]
-env_logger = "0.10"
+env_logger = "0.11"
 test-log = "0.2"
-simple_logger = "5.0"
-serde-big-array = "0.5"
+log = "0.4"
+tokio = { version = "1.20", features = ["macros", "rt-multi-thread"] }
```

Comment:
Since it's a dev-dependency it probably could just be "1" for the version.

Reply:
Good call... I actually think I'll just go ahead and make tokio a workspace dependency and switch everything to using that. We have way to many versions floating around.